code/modules/mob/living/simple_animal/hostile/nanotrasen.dm

```diff
@@ -56,5 +59,3 @@
 	rapid = 3
 	casingtype = /obj/item/ammo_casing/c46x30mm
 	projectilesound = 'sound/weapons/gunshot_smg.ogg'
-	loot = list(/obj/item/gun/ballistic/automatic/wt550,
```

Comment:
Can't do that, because the nanotrasen operatives that I based my hostiles entites off of for cybersun when then be dropping firearms which would be a substantial amount.